01-introduction.Rmd

```diff
@@ -8,10 +8,25 @@ By the end of the book you should be able to create new tools for geocomputation
 
 Over the last few decades free and open source software for geospatial data ('FOSS4G') has progressed at an astonishing rate (see [foss4g.org](http://foss4g.org/)).
 Thanks to FOSS4G and the wider open source movement geospatial analysis is no longer the preserve of those with expensive hardware and software: anyone can now download high performance spatial libraries on their computer.
+<!-- JM: I would delete this, since this is not really true.
 However, despite the growth of geospatial software that is *open source*, much of it is still not easy to script.
-Open source Geographic Information Systems (GIS) such as QGIS (see [qgis.org](http://qgis.org/en/site/)) have greatly reduced the 'barrier to entry' but their emphasis on the Graphical User Interface (GUI) can discourage reproducibility.
-This book focuses on the Command Line Interface (CLI), enabling reproducible, and 'computational' workflows, something we will expand on in Chapter 13.
+-->
+Open source Geographic Information Systems (GIS) such as QGIS (see [qgis.org](http://qgis.org/en/site/)) have greatly reduced the 'barrier to entry'.
+What is more, QGIS and other GIS software packages make command-line based processing easily accessible via application programming interfaces (API), something we will expand on in Chapter 13.
 <!--\@ref(gis) --><!-- REF NEEDS TO BE FIXED IN FUTURE-->
+Nevertheless, many users still prefer the Graphical User Interface (GUI) of GIS which in turn tends to discourage reproducibility (see Table \@ref(tab:gdsl)).
```

Comment:
I'm not sure that it's because they prefer the GUI, I think it's more that they've never had the opportunity to use a CLI with many slick features such as autocompletion and interactive panels for map viewing such as that provided by RStudio. I can tweak this after accepting the PR though so no biggy.